ros/lib/utils.py

```diff
@@ -109,3 +110,19 @@ def sort_io_dict(performance_utilization: dict):
 def system_ids_by_account(account_number):
     account_query = db.session.query(RhAccount.id).filter(RhAccount.account == account_number).subquery()
     return db.session.query(System.id).filter(System.account_id.in_(account_query))
+
+
+def create_performance_profile(session, system_id, fields):
+    """This method deletes an old entry from performance_profile &
+       inserts latest data inside performance_profile as well as
+       performance_profile_history table.
+    """
+    fields = {} if fields is None else fields
+    delete_record(session, PerformanceProfile, system_id=system_id)
+    get_or_create(
+        session, PerformanceProfile, 'system_id', **fields
+    )
+    get_or_create(
+        session, PerformanceProfileHistory,
+        ['system_id', 'report_date'], **fields
+    )
```

Comment:
@kgaikwad Shall we break this up into two separate functions?
Once a performance profile table is updated, function 1 can call function 2 to update performance profile history.
Suggesting this as it would easier to move to ORM events/signals in future, this way